src/jit/codegencommon.cpp

```diff
@@ -8519,6 +8519,13 @@ void                CodeGen::genFnProlog()
         genPrologPadForReJit();
         getEmitter()->emitMarkPrologEnd();
     }
+    
+#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING) && defined(FEATURE_SIMD)
+    // TODO-Cleanup: This logic can be implemented in
+    // genFnPrologCalleeRegArgs() for argument registers and 
+    // genEnregisterIncomingStackArgs() for stack arguments.
+    genClearStackVec3ArgUpperBits();
+#endif //FEATURE_UNIX_AMD64_STRUCT_PASSING && FEATURE_SIMD
```

Comment:
Similar TODO-64bit is comment is present in lower.cpp::LowerNode() for the case of GT_LCL_VAR/GT_STORE_LCL_VAR.  That too needs to be deleted.